src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/EnvironmentConfigurationHelper.cs

```diff
@@ -1,20 +1,45 @@
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
 namespace OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.Configuration
 {
     internal static class EnvironmentConfigurationHelper
     {
+        private static readonly Dictionary<IntegrationIds, Action<TracerProviderBuilder>> AddInstrumentation = new()
+        {
+            [IntegrationIds.HttpClient] = builder => builder.AddHttpClientInstrumentation(),
+            [IntegrationIds.AspNet] = builder => builder.AddSdkAspNetInstrumentation(),
+            [IntegrationIds.SqlClient] = builder => builder.AddSqlClientInstrumentation()
+        };
+
         public static TracerProviderBuilder UseEnvironmentVariables(this TracerProviderBuilder builder, Settings settings)
         {
             var resourceBuilder = ResourceBuilder
                 .CreateDefault()
                 .AddService(settings.ServiceName ?? "UNKNOWN_SERVICE_NAME", serviceVersion: settings.ServiceVersion);
 
-            return builder
+            builder
                 .SetResourceBuilder(resourceBuilder)
                 .SetExporter(settings);
```

Comment:
minor issue

```suggestion
            builder = builder
                .SetResourceBuilder(resourceBuilder)
                .SetExporter(settings);
```

this would be safer if for any reason in the future the builder would be immutable.

Probably it would never be the case. However, at least as a reader, I do not have to think if not assigning the value is a bug or not.

If you want to address that then you would need to also change `builder = addInstrumentation(builder);` and other places as well